addons/aws-efs-csi-driver/0.3.x/efs-1.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: aws-efs-csi-driver
+  labels:
+    kubeaddons.mesosphere.io/name: aws-efs-csi-driver
+    kubeaddons.mesosphere.io/provides: aws-efs-csi-driver
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.3.0-1"
+    appversion.kubeaddons.mesosphere.io/aws-efs-csi-drive: "0.3.0"
+    values.chart.helm.kubeaddons.mesosphere.io/aws-efs-csi-drive: "https://raw.githubusercontent.com/kubernetes-sigs/aws-efs-csi-driver/master/helm/values.yaml"
+spec:
+  namespace: kube-system
+  kubernetes:
+    minSupportedVersion: v1.15.6
+  cloudProvider:
+    - name: aws
+      enabled: true
+  chartReference:
+    chart: aws-efs-csi-driver
+    repo: https://github.com/kubernetes-sigs/aws-efs-csi-driver
+    version: 0.3.0
```

Comment:
```suggestion
    version: 0.3.0
    values: |
      sidecars:
        livenessProbeImage:
          tag: "v2.0.0"
        nodeDriverRegistrarImage:
          tag: "v1.2.0"
      node:
        # add annotations for e.g. kube2iam or kiam usage here, or other purposes
        podAnnotations: {}
        tolerations:
        - effect: NoSchedule
          operator: Exists
        - effect: NoExecute
          operator: Exists
        - key: CriticalAddonsOnly
          operator: Exists
```

Reply:
maybe nodeDriverRegistrarImage v1.2.0 is not working with the efs-plugin, but we will check and change accordingly
CSI details here btw :) https://kubernetes-csi.github.io/docs/